samuelbwambale/my-diary-react-APP
# #162439714 Add all entries to dashboard

### What does this PR do?

Enables the user to view all their entries after logging in

#### Description of Task to be completed?

Currently, the user can not see their entries

This feature adds a list of users entries on the dashboard. The list is viewed after login in.

#### How should this be manually tested?

Pull the changes on a new branch and run `npm install`
run `npm start` to start the server
View `http://localhost:3000/` and click on login nav link
Enter the email and password used when creating the account. If credentials are valid, you will be redirected to the dashboard. 
On the dashboard, the list of entries is displayed. 

#### What are the relevant pivotal tracker stories?

- [#162439714](https://www.pivotaltracker.com/story/show/162439714)

#### Screenshots (if appropriate)


## Checklist: I completed these to help reviewers
